src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -68,48 +68,54 @@ <h4 class="panel-title border-bottom-small" translate>
         <h4 class="panel-title border-bottom-small" translate>
           {{'GIFT_AMOUNT'}}
         </h4>
-        <div class="panel-body pt0">
+        <div class="panel-body p0">
           <div ng-if="$ctrl.useSuggestedAmounts">
-            <div class="radio radio-custom-amount"
-                 ng-repeat="suggested in $ctrl.suggestedAmounts | orderBy:'order'">
-              <label>
-                <input name="suggestedAmount"
-                       type="radio"
-                       ng-click="$ctrl.changeAmount(suggested.amount)"
-                       ng-checked="!$ctrl.customInputActive && $ctrl.itemConfig.AMOUNT === suggested.amount" />
-                {{$ctrl.suggestedAmount(suggested.amount)}} {{suggested.label}}
-              </label>
-            </div>
-            <div class="radio radio-custom-amount form-inline" ng-class="{'has-error': ($ctrl.itemConfigForm.amount | showErrors)}">
-              <div class="form-group">
+            <div class="button-group">
+              <div 
+                  role="button"
+                  class="suggested-button"
+                  ng-click="$ctrl.changeAmount(suggested.amount)"
+                  ng-class=" {'active': !$ctrl.customInputActive && $ctrl.itemConfig.AMOUNT === suggested.amount}"
+                  ng-repeat="suggested in $ctrl.suggestedAmounts | orderBy:'order'">
+                  <div class="amount-box">
+                    {{$ctrl.suggestedAmount(suggested.amount)}}
+                  </div>
+                  <div class="label-text">
+                    {{suggested.label}}
+                  </div>
```

Comment:
How well does this work with a screen reader?

Reply:
<img width="514" alt="Screenshot 2024-11-27 at 11 57 38 AM" src="https://github.com/user-attachments/assets/989e73bd-3180-4c93-a8d5-2f2f049db2af"><br>
Each button and button label text is in the Accessibility Tree, but I have not tested it myself using a screen reader extension.

Reply:
We'll need to change the element type to radio input and I can just style it to look as it does now. Also, the custom amount isn't clearly showing up in the Accessibility Tree, so I'll make sure that works correctly.